_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -0,0 +1,187 @@
+7. Click `Launch Instance`
+8. On **EC2 Dashboard** select the new created instance and click `Security`
+9. On **Security Groups**, click on the security group
+10. On **Inbound Rules** tab, click `Edit Inbound Rules`
+11. Click `Add rule` and select `PostgreSQL` from the dropdown menu and `Anywhere-IPv4`.
+12. Click `Save rules`
+13. On **EC2 Dashboard** select the new created instance and click `Connect`
+14. Click `SSH Client` and follow the instructions
+
+Once logged into the ec2 instance we perform the following tasks:
+
+### Install Libraries and Dependencies
+Update the local package manager `apt` 
+```bash
+sudo apt update
+```
+Install [NodeJS](https://nodejs.org)
+```bash
+sudo apt install nodejs
+```
+Check the install was ok, you should see the version installed.
+```bash
+node -v
+```
+
+
+Install [npm](https://www.npmjs.com)
+```bash
+sudo apt install npm
+```
+
+
+
+Install [yarn](https://yarnpkg.com)
+```bash
+sudo npm install yarn –g
+```
+
+### Install PostgreSQL Server
+
+Install [PostgreSQL](https://www.postgresql.org)
+```bash
+sudo apt-get -y install postgresql
+```
+Once is installed, create a password for the user `postgres`
+
+```bash
+sudo su postgres
+```
+
+```bash
+psql
+```
+
+```bash
+ALTER USER postgres password '<MY_PASSWORD>';
+```
+
+Quit psql typing `\q`
+
+Exit postgres user typing `exit`
+
+Navigate to main folder inside postgresql/version/
+```bash
+cd /etc/postgresql/14/main/
+```
+We need to edit two files, `pg_hba.conf` and `postgresql.conf`
+```bash
+sudo nano pg_hba.conf
+```
+Scroll down the file and Add `host, all, all, 0.0.0.0/0, md5`, has to be the first line before `local, all, postgres, , peer`
+
+| TYPE | DATABASE | USER | ADDRESS | METHOD |
+| ---- | -------- | ---- | ------- | ------ |
+| host | all | all | 0.0.0.0/0 | md5 |
+| local | all | postgres |  | peer |
+
+
+```bash
+sudo nano postgresql.conf
+```
+Search for `#listen_addresses='localhost'`, uncomment the line and replace `localhost` for `*`
+
+Restart the PostgreSQL server
+```bash
+sudo service postgresql restart
+```
+
+### Setup Parse Server
+
+Create a directory
+```bash
+cd ~
+```
+```bash
+mkdir parse-server
+```
+```bash
+cd parse-server
+```
+
+Run the bash script and follow the instructions, the script have some visual issues and the keys generation doesn't work.
+```bash
+sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
+```
+After that, we need to setup the configuration file, use your own `appId`, `masterKey` and `clientKey`, use random strings or some generator tool to create secured keys.
+```bash
+sudo nano -w config.json
+```
+This are the basic options of the config.json file, for the full list you can type `parse-server --help` or refer to the [Full Options Document](http://parseplatform.org/parse-server/api/master/ParseServerOptions.html) for more details.
+```js
+{
+  "appId": "<APP_ID>",
```

Comment:
I think we could use the same values as below, `exampleMasterKey`, etc

Reply:
done, I also updated the Full Options Document link to point the last version of the file